.azure/pipelines/ci.yml

```diff
@@ -7,6 +7,7 @@ trigger:
   batch: true
   branches:
     include:
+    - blazor-wasm
```

Comment:
Should this be undone?

Reply:
Doesn't matter either way. AzDO will load the YAML for the branch that's being built.